rskj-core/src/test/java/co/rsk/net/sync/SnapSyncStateTest.java

```diff
@@ -219,10 +225,86 @@ void givenOnSnapStateChunkIsCalled_thenJobIsAddedAndRun() throws InterruptedExce
         assertEquals(msg, jobArg.getValue().getMsg());
     }
 
```

Comment:
question:

What's the benefit of test the set and get? Do they have something different than just set the variable?

Reply:
We can improve that in a future ticket. I am closing the threads now.